src/components/datepicker/js/datepickerDirective.js

```diff
@@ -402,9 +402,17 @@
 
     // Responds to external changes to the model value.
     self.ngModelCtrl.$formatters.push(function(value) {
+      var parsedValue = angular.isDefined(value) ? Date.parse(value) : null;
+
+      if (parsedValue) {
```

Comment:
You should change this to `parsedValue == null || isNaN(parsedValue)` or else "1970-01-01T00:00:00.000Z" won't be validated